java/maven/test/unit/src/org/netbeans/modules/maven/execute/DefaultReplaceTokenProviderTest.java

```diff
@@ -64,23 +65,71 @@ public void testTestSingle() throws Exception {
         // XXX test src/main/java selections
         // XXX test selections across groups, or outside groups
         // XXX test single methods
-        
+
         //#213671
         assertEquals(null, ActionProviderImpl.replacements(p, ActionProvider.COMMAND_RUN, Lookup.EMPTY).get(DefaultReplaceTokenProvider.PACK_CLASSNAME));
         assertEquals(null, ActionProviderImpl.replacements(p, ActionProvider.COMMAND_RUN, Lookup.EMPTY).get(DefaultReplaceTokenProvider.CLASSNAME));
         assertEquals(null, ActionProviderImpl.replacements(p, ActionProvider.COMMAND_RUN, Lookup.EMPTY).get(DefaultReplaceTokenProvider.CLASSNAME_EXT));
-        
+
     }
-    
+
     public void testNgSingle() throws Exception {
         TestFileUtils.writeFile(d, "pom.xml", "<project><modelVersion>4.0.0</modelVersion>"
                 + "<groupId>g</groupId><artifactId>a</artifactId><version>0</version></project>");
         TestFileUtils.writeFile(d, "src/test/java/p1/FirstNGTest.java", "package p1; class FirstNGTest {}");
         TestFileUtils.writeFile(d, "src/main/java/p1/First.java", "package p1; class First {}");
-        
+
         Project p = ProjectManager.getDefault().findProject(d);
         assertEquals("p1.FirstNGTest", ActionProviderImpl.replacements(p, ActionProvider.COMMAND_TEST_SINGLE, Lookups.singleton(d.getFileObject("src/main/java/p1/First.java"))).get(DefaultReplaceTokenProvider.PACK_CLASSNAME));
 
     }
 
+    public void testIntegration2Files() throws Exception {
```

Comment:
could you also add test that produces multiple patterns in `packageClassName` ?

Reply:
multiple patterns ... as to run 1 test class and 1 package as `-Dtest=some.it.pkg.other.It3Test,some.it.pkg.**` ... or I misunderstood what you suggested.

Reply:
seems covered by `testIntegration2FilesAndPacakge`